zerver/lib/export.py

```diff
@@ -780,22 +780,22 @@ def get_realm_config() -> Config:
         model=Stream,
         exclude=["email_token"],
         normal_parent=realm_config,
-        parent_key="realm_id__in",
+        foreign_key="realm",
     )
 
     stream_recipient_config = Config(
         table="_stream_recipient",
         model=Recipient,
         normal_parent=stream_config,
-        parent_key="type_id__in",
+        foreign_key="type",
```

Comment:
We have two unrelated fields, `type` and `type_id`, on this table, so this feels at least a code smell.

Reply:
I wonder if another name that's more like `include_rows="type_id__in"` would be a better readability improvement.